pkg/controller/upgradeconfig/cluster_upgrader.go

```diff
@@ -549,7 +551,7 @@ func (cu ClusterUpgrader) UpgradeCluster(c client.Client, m maintenance.Maintena
 func performClusterHealthCheck(c client.Client) (bool, error) {
 	sa := &corev1.ServiceAccount{}
 
-	err := c.Get(context.TODO(), types.NamespacedName{"openshift-monitoring", "prometheus-k8s"}, sa)
+	err := c.Get(context.TODO(), types.NamespacedName{Namespace: "openshift-monitoring", Name: "prometheus-k8s"}, sa)
 	if err != nil {
 		return false, err
```

Comment:
As per my comment on line 594:
```suggestion
		return false, fmt.Errorf("Unable to fetch prometheus-k8s service account: %s", err)
```

Reply:
Thanks a lot for the review Matt ! Have submitted changes..